src/main/telemetry/telemetry.h

```diff
@@ -37,7 +37,7 @@ typedef enum {
 
 typedef struct telemetryConfig_s {
     uint8_t telemetry_switch;               // Use aux channel to change serial output & baudrate( MSP / Telemetry ). It disables automatic switching to Telemetry when armed.
-    serialInversion_e telemetry_inversion;      // also shared with smartport inversion
+    uint8_t telemetry_inversion;            // also shared with smartport inversion
```

Comment:
Port remap is one.

It works great for blackbox on NAZE - UART1 is connected to USB normally, but after arming pins are remapped (to MOTOR3/MOTOR4 servo headers). It is much easier and cleaner to attach logger here (and avoids MSP frames in blackbox as bonus).

It would work similarly for telemetry ...

And singlewire mode for telemetry would be another candidate.